src/fragments/lib/auth/flutter/signin_web_ui/20_platform_specific_setup.mdx

```diff
@@ -92,3 +68,50 @@ Add the following entry to the URL scheme in the `Info.plist` file in your app's
      <!-- ... -->
      </dict>
 ```
+</Block>
+
+<Block name="Developer Preview (Mobile, Web & Desktop)">
+
+## Web Platform Setup
+
+To use Hosted UI in your Flutter web application locally, you must run the app with the `--web-port=3000` argument (with the value being whichever port you assigned to localhost host when configuring your redirect URIs).
+
+## Android Platform Setup
+
+Add the following `queries` element to the `AndroidManifest.xml` file in your app's `android/app/src/main` directory, as well as the following `intent-filter` to the `MainActivity` in the same file.
+
+Replace `myapp` with your redirect URI scheme as necessary:
+
+```xml
+<queries>
+    <intent>
+        <action android:name=
+            "android.support.customtabs.action.CustomTabsService" />
+    </intent>
+</queries>
+<application>
+  ...
+  <activity
+        android:name=".MainActivity">
+        <intent-filter>
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.DEFAULT" />
+            <category android:name="android.intent.category.BROWSABLE" />
+            <data android:scheme="myapp" />
+        </intent-filter>
+  </activity>
+  ...
+</application>
+```
+
+## macOS
+
+If you haven't already, open XCode and enable the App Sandbox capability and then select "Outgoing Connections (Client)" under "Network".
+
+## iOS, Windows and Linux Setup
+
+No specific platform configuration is required. However, please note that on Desktop applications an attempt by the user to cancel the sign in (for example, by closing their browser) will not automatically abort the operation. You may need to expose a 'cancel' operation in your UI to an appropriate interface and state.
```

Comment:
Should the desktop note here be a callout maybe? sounds like something we are trying to highlight that might get buried in the paragraph.

Reply:
This would apply for other sections where we use this language

Reply:
We will fix this before release. Let's remove